pym/bob/generators/QtCreatorGenerator.py

```diff
@@ -109,8 +109,10 @@ def addBuildSteps(outFile, buildMeFile):
     addBuildConfig(outFile, 2, "Bob dev (no checkout)", "-b", buildMeFile)
     addBuildConfig(outFile, 3, "Bob dev (no deps)", "-n", buildMeFile)
     addBuildConfig(outFile, 4, "Bob dev (no checkout, no deps)", "-nb", buildMeFile)
+    addBuildConfig(outFile, 5, "Bob dev (no checkout, Debug)", "-b -DCMAKE_BUILD_TYPE='DEBUG'", buildMeFile)
```

Comment:
this makes no sense for non cmake projects. There is already another PR for this. https://github.com/BobBuildTool/bob/pull/78 

Please add the clean build config to eclipse also.